examples/graphql-service-query-complexity/graphql_service_query_complexity.metatags

```diff
@@ -0,0 +1,2 @@
+description: This example demonstrates securing a GraphQL service query complexity analysis.
+keywords: ballerina, ballerina by example, bbe, graphql, service, security, query complexity
```

Comment:
Shouldn't it be
```suggestion
description: This example demonstrates GraphQL service query complexity analysis.
keywords: ballerina, ballerina by example, bbe, graphql, service, query complexity
```
?

Reply:
Query complexity analysis is used to secure a GraphQL service.